lua/nordic/init.lua

```diff
@@ -13,15 +13,13 @@ function M.load(opts)
         require 'nordic.config'.setup(opts)
     end
 
-    -- Setup colors.
-    local colors = require('nordic.colors')
-    colors.apply_modifications()
-    colors.extend_palette()
-
-    -- Apply theme.
-    local G = require 'nordic.groups'
-    U.highlight(G.get_groups())
-    G.set_term_colors()
+    -- Setup colors
+    require('nordic.colors').extend_palette()
+
+    -- Apply theme
+    local groups = require 'nordic.groups'
```

Comment:
The current convention (in this codebase at least) is to name modules with a capital letter.  So this should be `G`.  Not a big deal, just a stylistic choice.

Reply:
Ok I though capital letters were for globals I will change this.

Reply:
Actually a better idea.  But we can change that in a different PR

Reply:
Lets just leave it for now, that is actually the reason I wrote it like this.  Catppuccin also does this.

Reply:
Remember to change this to G.